### PR TITLE
Initial code check in. Copied from w3c/PFE-analysis.

### PR DESCRIPTION
**Pull request by andyjgf**

### CODE REVIEW

File: analysis/pfe_methods/unicode_range_data/slicing_strategy_loader.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+"""Helper that can load slicing strategy protos by name."""
```
Comment (garretrieger) [inline analysis/pfe_methods/unicode_range_data/slicing_strategy_loader.py:1]: I believe you can omit this file.

File: analysis/pfe_methods/unicode_range_data/slicing_strategy_loader_test.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
+"""Unit tests for the slicing_strategy_loader module."""
```
Comment (garretrieger) [inline analysis/pfe_methods/unicode_range_data/slicing_strategy_loader_test.py:1]: Same here.

File: WORKSPACE [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+http_archive(
+    name = "gtest",
+    sha256 = "94c634d499558a76fa649edb13721dce6e98fb1e7018dfaeba3cd7a083945e91",
+    strip_prefix = "googletest-release-1.10.0",
+    url = "https://github.com/google/googletest/archive/release-1.10.0.zip",
+)
+
+# FontTools
+http_archive(
+    name = "fonttools",
```
Comment (garretrieger) [inline WORKSPACE:15]: This shouldn't be needed if you remove the two python files I mentioned below.

**Review by garretrieger — APPROVED** [review]
It would be good to also bring over the formatting scripts and the CI setup, but you can do that in a seperate PR.

File: WORKSPACE (hunk 3 above)
Comment (andyjgf) [reply WORKSPACE]: Done

File: analysis/pfe_methods/unicode_range_data/slicing_strategy_loader_test.py (hunk 2 above)
Comment (andyjgf) [reply analysis/pfe_methods/unicode_range_data/slicing_strategy_loader_test.py]: Done.

File: analysis/pfe_methods/unicode_range_data/slicing_strategy_loader.py (hunk 1 above)
Comment (andyjgf) [reply analysis/pfe_methods/unicode_range_data/slicing_strategy_loader.py]: Done.

**Pull request merged**